core/src/test/java/kafka/log/remote/RemoteLogManagerTest.java

```diff
@@ -735,7 +781,7 @@ void testMetricsUpdateOnCopyLogSegmentsFailure() throws Exception {
         verify(remoteStorageManager, times(1)).copyLogSegmentData(any(RemoteLogSegmentMetadata.class), any(LogSegmentData.class));
 
         // Verify we should not have updated the highest offset because of write failure
-        verify(mockLog, times(0)).updateHighestOffsetInRemoteStorage(anyLong());
+        verify(mockLog, times(1)).updateHighestOffsetInRemoteStorage(anyLong());
```

Comment:
```suggestion
        verify(mockLog).updateHighestOffsetInRemoteStorage(anyLong());
```
times(1) is implicit